src/mgr/PyModules.cc

```diff
@@ -371,13 +373,19 @@ int PyModules::init()
       // or the right thread state (this is deliberate).
       derr << "Error loading module '" << module_name << "': "
         << cpp_strerror(r) << dendl;
+      failed_modules.push_back(module_name);
       // Don't drop out here, load the other modules
     } else {
       // Success!
       modules[module_name] = std::move(mod);
     }
   }
 
+  if (!failed_modules.empty()) {
+    clog->error() << "Failed to load manager modules: " << joinify(
```

Comment:
MGR would not be correct (since it's not an acronym), but I've changed it to "ceph-mgr modules" to be completely unambiguous.